### PR TITLE
changed paymentProviderCustomer to an array

### PR DESCRIPTION
**Pull request by roosnic1**

### CODE REVIEW

**Review by penk85 — DISMISSED** [review]
:+1:

File: packages/api-db-mongodb/src/migration.ts [hunk 1]
```diff
@@ -613,6 +614,33 @@ export const Migrations: Migration[] = [
...
+      const users = await db.collection(CollectionName.Users)
+
+      const allUsers: DBUser[] = await users.find({}).toArray()
+
+      for (const user of allUsers) {
+        const paymentProvidersCustomersArray: PaymentProviderCustomer[] = []
+        const paymentProviderCustomers = Object.keys(user.paymentProviderCustomers)
+        paymentProviderCustomers.forEach(ppc => {
+          // @ts-ignore
+          const temp = user.paymentProviderCustomers[ppc]
```
Comment (camilleory) [inline packages/api-db-mongodb/src/migration.ts:631]: the name of the `temp `variable is not very descriptive, could it be more self-explanatory or can you explain what it describes?

File: packages/api/src/db/user.ts [hunk 2]
```diff
@@ -117,8 +117,8 @@ export interface DeleteUserSubscriptionArgs {
 }
 
 export interface PaymentProviderCustomer {
-  readonly id: string
-  readonly createdAt: Date
```
Comment (camilleory) [inline packages/api/src/db/user.ts:121]: Is `createdAt` not relevant anymore ?

**Review by camilleory — CHANGES_REQUESTED** [review]

File: packages/api/src/db/user.ts (hunk 2 above)
Comment (roosnic1) [reply packages/api/src/db/user.ts:121]: Yeah i added it in the past but realized now that it was premature optimization and we don't need it.

**Review by camilleory — APPROVED** [review]

**Review by penk85 — APPROVED** [review]

**Review by lakeside763 — APPROVED** [review]

**Review by bishoymelek — COMMENTED** [review]
Does this has anything to do with the Editor or Public API?

**Pull request merged**